src/WebServer.ino

```diff
@@ -2984,21 +2984,16 @@ void handle_json()
   if (tasknr.length() == 0)
   {
     reply += F("{\"System\":{\n");
-    reply += F("\"Build\":");
-    reply += BUILD;
-    reply += F(",\n\"Git Build\":\"");
-    reply += BUILD_GIT;
-    reply += F("\",\n\"Unit\":");
-    reply += Settings.Unit;
-    reply += F(",\n\"Local time\":");
-    reply += F("\"");
-    reply += getDateTimeString('-',':',' ');
-    reply += F("\"");
-    reply += F(",\n\"Uptime\":");
-    reply += wdcounter / 2;
-    reply += F(",\n\"Free RAM\":");
-    reply += ESP.getFreeHeap();
-    reply += F("\n},\n");
+
+	reply += F("\"Name\":");		reply += F("\"");	reply += Settings.Name;					reply += F("\"");	reply += F(",");
```

Comment:
did you test if this generates valid json? You're missing at least a quote and a comma here and on other lines..

Reply:
I've of course tested b4 doing the PR, but too be sure , I've just uploaded it today again. And yes it generate a perfect json format.

_BTW didn't you fully scroll the code line to the right, when you reviewed it ? ;-)_

Reply:
_BTW  On GitHub, when you review the changes in "Split" view you can NOT scroll to right, while if you select the "Unified" view , you can now scroll to right, to see the full lines_

Reply:
ohhh right..i see it now. is there a reason for this unconventional coding style?